content/en/ddsql_editor/_index.md

```diff
@@ -68,9 +68,14 @@ Browse and re-run saved queries in the side panel.
 
 {{< img src="ddsql_editor/saved-queries-panel.png" alt="A list of saved queries" style="width:60%;" >}}
 
+## Permissions
+
+To access the DDSQL Editor app, users need the `ddsql_editor_read` permission. This permission is included in the Datadog Standard Role by default. If your organization uses custom roles, add this permission to the appropriate role. For more information on managing permissions, see the [RBAC documentation][3].
```

Comment:
```suggestion
To access the DDSQL Editor app, users need the `ddsql_editor_read` permission. This permission is included in the Datadog Read Only Role by default. If your organization uses custom roles, add this permission to the appropriate role. For more information on managing permissions, see the [RBAC documentation][3].
```

Hey @estherk15! Circled back with the team, and we've decided to place it into the `Read Only` role instead.

We'll have to make some code changes that may take a while. I will stamp the approval when the doc change is ready to go live. Thanks again!

Reply:
Sounds good @wbew, I'll put a Do Not Merge label. When this is good to go, feel free to remove and comment `/merge`!